src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIError.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+namespace System.Data.SqlClient.SNI
+{
+    /// <summary>
+    /// SNI error
+    /// </summary>
+    internal class SNIError
+    {
+        public SNIProviders provider;
+        public String errorMessage;
+        public uint nativeError;
+        public uint sniError;
+        public String function;
+        public uint lineNumber;
+
+        public SNIError(SNIProviders provider, uint nativeError, uint sniErrorCode, String errorMessage)
```

Comment:
Nit: we're not yet consistent across the whole codebase, but we're trying to use the C# keywords instead of type names where possible, e.g. string instead of String